src/code/z_play.c

```diff
@@ -943,22 +953,27 @@ void Play_Update(PlayState* this) {
                 LOG_NUM("1", 1, "../z_play.c", 3677);
             }
 
-            if (this->unk_1242B != 0) {
+            // Update viewpoint
```

Comment:
similarly I think most of this is self explanatory. I think the last comment about the weird ^ 3 is the only thing that needs explaining